### ISSUE TITLE
Running plugins fails on client

### ISSUE THREAD

**Issue by brollb** [webgme/webgme#1104]
Without seeing any other visible errors, I am seeing:
```
logger.js:66Error network status, no actions done UNCAUGHT_EXCEPTIONo.error @ logger.js:66
InterpreterManager.js:32Uncaught TypeError: Cannot read property '_client' of undefined
```

I am guessing it could be related to the automerge issues but am not sure of the cause.

**Comment by brollb** [comment]
![screen shot 2016-08-18 at 2 04 38 pm](https://cloud.githubusercontent.com/assets/4982789/17786932/c428b5dc-654c-11e6-836f-6e14a36b37cd.png)

**Comment by brollb** [comment]
wait, I see this has been fixed in 59894aa
